WebServices/TodoWCF/TodoWCF/TodoWCF.csproj

```diff
@@ -1,26 +1,127 @@
+    <EmbeddedResource Include="Views\TodoItemPage.xaml">
       <Generator>MSBuild:UpdateDesignTimeXaml</Generator>
+      <SubType>Designer</SubType>
     </EmbeddedResource>
-    <EmbeddedResource Update="Views\TodoItemPage.xaml">
+    <EmbeddedResource Include="Views\TodoListPage.xaml">
       <Generator>MSBuild:UpdateDesignTimeXaml</Generator>
+      <SubType>Designer</SubType>
     </EmbeddedResource>
   </ItemGroup>
-
+  <ItemGroup>
+    <Compile Include="Connected Services\TodoWCFService\Reference.cs">
+      <AutoGen>True</AutoGen>
+      <DesignTime>True</DesignTime>
+      <DependentUpon>Reference.svcmap</DependentUpon>
+    </Compile>
+    <Compile Include="Data\ISoapService.cs" />
+    <Compile Include="Data\SoapService.cs" />
+    <Compile Include="Data\TodoItemManager.cs" />
+    <Compile Include="ITextToSpeech.cs" />
+    <Compile Include="Models\TodoItem.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="TodoWCF.cs" />
+    <Compile Include="Views\TodoItemPage.xaml.cs">
+      <DependentUpon>TodoItemPage.xaml</DependentUpon>
+    </Compile>
+    <Compile Include="Views\TodoListPage.xaml.cs">
+      <DependentUpon>TodoListPage.xaml</DependentUpon>
+    </Compile>
+  </ItemGroup>
+  <ItemGroup>
+    <WCFMetadata Include="Connected Services\" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="Connected Services\TodoWCFService\TodoService.wsdl" />
+    <None Include="Connected Services\TodoWCFService\TodoService.xsd">
+      <SubType>Designer</SubType>
+    </None>
+    <None Include="Connected Services\TodoWCFService\TodoService1.xsd">
+      <SubType>Designer</SubType>
+    </None>
+    <None Include="Connected Services\TodoWCFService\TodoService2.xsd">
+      <SubType>Designer</SubType>
+    </None>
+    <None Include="Connected Services\TodoWCFService\TodoWCF.TodoWCFService.TodoItem.datasource">
+      <DependentUpon>Reference.svcmap</DependentUpon>
+    </None>
+    <None Include="packages.config" />
+    <None Include="Properties\ServiceReferences.Designer.ClientConfig" />
+  </ItemGroup>
+  <ItemGroup>
+    <WCFMetadataStorage Include="Connected Services\TodoWCFService\" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="Connected Services\TodoWCFService\TodoService.disco" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="Connected Services\TodoWCFService\configuration91.svcinfo" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="Connected Services\TodoWCFService\configuration.svcinfo" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="Connected Services\TodoWCFService\Reference.svcmap">
+      <Generator>WCF Proxy Generator</Generator>
+      <LastGenOutput>Reference.cs</LastGenOutput>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="Xamarin.Forms.Core, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\Xamarin.Forms.3.6.0.264807\lib\netstandard1.0\Xamarin.Forms.Core.dll</HintPath>
```

Comment:
Can you use 3.6.0.220655 instead? All samples in the repo are on that version, which makes it easy to do a find/replace for future mass updates.